chef/cookbooks/provisioner/files/default/crowbar_join.init.suse

```diff
@@ -4,7 +4,7 @@
 # Required-Start:    $syslog $network $remote_fs sshd
 # Should-Start:      neutron-ovs-cleanup xend libvirtd
 # Required-Stop:     $syslog $network $remote_fs sshd
-# Should-Stop:       openvswitch-switch ovs-usurp-config-br-fixed ovs-usurp-config-br-public
+# Should-Stop:       ovs-usurp-config-br-fixed ovs-usurp-config-br-public
```

Comment:
I think the service is gone as well, at least it looks that way on my SP3 node. But I can remove this line change and keep just the package one